

message setup_msg {
// <注册>消息
    required string log_id = 1;	//用户名
    required string pwd = 2;	//密码(加密之后的)
    required string email = 3;	//邮箱
};


message up_usrinfo_msg {
//  <更新用户信息>消息
    required string log_id = 1;		//用户名
    optional string nick_name = 2;	//昵称
    optional string name = 3;		//姓名
    optional string sex = 4;			//性别
    optional string birth = 5;		//生日
    optional string hometown = 6;	//故乡
    optional string location = 7;		//所在地
    optional string blood = 8;		//身型
    optional string tel = 9;			//电话
    optional int32 job = 10;		//职业
    optional string school = 11;		//学校
};